### PR TITLE
Remove most cases of py3-conditional code using PY3 from six

### PR DESCRIPTION
**Pull request by foolip**
The only remaining case is more complicated and handled separately:
https://github.com/web-platform-tests/wpt/pull/28740

### CODE REVIEW

File: html/semantics/forms/form-submission-0/resources/file-submission.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
 from wptserve.utils import isomorphic_decode
 
 def main(request, response):
     testinput = request.POST.first(b"testinput")
-    if PY3:
-        # The test asserts the string representation of this FieldStorage
-        # object, but unfortunately the value property has different types in
-        # Python 2 and 3. Unify them to native strings.
-        testinput.value = isomorphic_decode(testinput.value)
+    # The test asserts the string representation of this FieldStorage
```
Comment (inexorabletash) [inline html/semantics/forms/form-submission-0/resources/file-submission.py:5]: Comment no longer needed?

**Review by pwnall — APPROVED** [review]
LGTM w/ inexorabletash@ comment addressed

File: html/semantics/forms/form-submission-0/resources/file-submission.py (hunk 1 above)
Comment (foolip) [reply html/semantics/forms/form-submission-0/resources/file-submission.py:5]: Yeah, I'll remove it.

Comment (annevk) [reply html/semantics/forms/form-submission-0/resources/file-submission.py:5]: @foolip it seems you forgot to remove the comment.

Comment (foolip) [reply html/semantics/forms/form-submission-0/resources/file-submission.py:5]: Hmm, I definitely made the change, but maybe it's left sitting on my computer. Will fix!

Comment (foolip) [reply html/semantics/forms/form-submission-0/resources/file-submission.py:5]: I went too far: https://github.com/web-platform-tests/wpt/pull/28758

**Pull request merged**